examples/grpc-client/main.go

```diff
@@ -37,6 +40,34 @@ func run() error {
 
 	client := runnerv1.NewRunnerServiceClient(conn)
 
+	if *createSession {
+		resp, err := client.CreateSession(context.Background(), &runnerv1.CreateSessionRequest{
+			Envs: os.Environ(),
+		})
+		if err != nil || resp.Session == nil {
+			return err
+		}
+
+		_, _ = fmt.Println(resp.Session.Id)
+
+		return nil
+	}
+
+	if *deleteSession != "" {
+		id := *deleteSession
+
+		_, err := client.DeleteSession(context.Background(), &runnerv1.DeleteSessionRequest{
+			Id: id,
+		})
+		if err != nil {
+			return err
+		}
+
+		_, _ = fmt.Printf("Successfully deleted session \"%s\"", id)
```

Comment:
```suggestion
		_, _ = fmt.Printf("Successfully deleted session %q", id)
```

`%q` will quote the value.